front/app/modules/commercial/flag_inappropriate_content/admin/components/ActivityWarningsTab.tsx

```diff
@@ -15,22 +15,29 @@ declare module 'modules/commercial/moderation/admin/containers/index' {
 
 interface Props {
   onData: (data: InsertConfigurationOptions<ITabItem>) => void;
+  activeFlagsCount: number;
 }
 
 const ActivityWarningsTab = ({
   onData,
+  activeFlagsCount,
   intl: { formatMessage },
 }: Props & InjectedIntlProps) => {
   useEffect(
     () =>
       onData({
         configuration: {
           name: 'warnings',
-          label: formatMessage(messages.warnings),
+          label: (
+            <span>
+              {`${formatMessage(messages.warnings)} (${activeFlagsCount})`}
+            </span>
+          ),
         },
         insertAfterName: 'read',
+        reinsertAfterUpdate: true,
       }),
-    []
+    [activeFlagsCount]
```

Comment:
Seems like you are missing onData as a dependency?

Reply:
This insertion mechanism doesn't work if onData is a dependency. I'm not sure why exactly, but we can't add it because the tabs block completely.

Reply:
Hm, this is a strong indicator that there is a problem with the way we manage state and props here. 
We can leave it as is for now but the issue will come back around with the linter update and we do have to fix these things at some point.